Authentication/AuthenticationBootstrapper.swift

```diff
@@ -162,7 +162,7 @@ public class AuthenticationBootstrapper<User: UserType, SessionService: SessionS
     }
     
     public func createSignupView() -> SignupViewType {
-        let view = SignupView.loadFromNib()
+        let view = SignupView.loadFromNib()!
```

Comment:
I think we should handle the nil case here and raise an exception and log an error explaining what the problem might be and how it should be solved. Either that or make this method return an optional